### PR TITLE
Fix bugs related to dates, wording, and country select

### PR DESCRIPTION
**Pull request by bryce-mcmath**
### Please check if the PR fulfills these requirements:

- [x] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
Bug fixes

### CODE REVIEW

File: jha/src/views/enrolment/PersonalInfoPage.vue [hunk 1]
```diff
@@ -343,9 +343,8 @@
                       && !$v.moveFromOrigin.required"
                     aria-live="assertive">Jurisdiction of origin is required.</div>
                   <div class="text-danger"
-                    v-if="$v.moveFromOrigin.$dirty
-                      && !$v.moveFromOrigin.nonCanadaValidator"
-                    aria-live="assertive">Jurisdiction of origin cannot be Canada.</div>
+                      v-if="moveFromOrigin === 'Canada'"
```
Comment (stephanieharrison-maximus) [inline jha/src/views/enrolment/PersonalInfoPage.vue:346]: I can see you removed the v-if that checks if the validation is dirty or not. Is that intentional? All of the rest seem to have it.

File: jha/src/views/enrolment/PersonalInfoPage.vue [hunk 2]
```diff
@@ -343,9 +343,8 @@
...
+                      aria-live="assertive">Jurisdiction of origin cannot be Canada.</div>
```
Comment (harrymaynard-maximus) [inline jha/src/views/enrolment/PersonalInfoPage.vue:347]: I remember changing this a couple weeks ago. I'd prefer to stick with using the validator result for this error message. It reduces logic duplication, and makes debugging error messages easier. What was the reason for the update?

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: jha/src/views/enrolment/PersonalInfoPage.vue (hunk 2 above)
Comment (bryce-mcmath) [reply jha/src/views/enrolment/PersonalInfoPage.vue:347]: Unlike the other fields CountrySelect has no blur handler, so if they select Canada they won't know anything is wrong until they click continue. Checking the condition directly makes the error appear as soon as they select Canada. The same pattern is used on the spouse page. I can change them both to just use the validator check if you like, doesn't make much difference to me

File: jha/src/views/enrolment/PersonalInfoPage.vue (hunk 1 above)
Comment (bryce-mcmath) [reply jha/src/views/enrolment/PersonalInfoPage.vue:346]: See comment under Harry's requested changes

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good.

**Review by stephanieharrison-maximus — COMMENTED** [review]
Looks good!

**Pull request merged**